source/ncengine/graphics2/diligent/resource/ShaderBindings.h

```diff
@@ -26,12 +26,13 @@ class ShaderBindings
                 context, device,
                 "PerFrameResourceSignature",
                 0,
-                StructuredBufferResourceDesc{"MeshRendererBufferData",     Diligent::SHADER_TYPE::SHADER_TYPE_VS_PS, maxMeshRenderers,     maxMeshRenderers},
-                StructuredBufferResourceDesc{"DirectionalLightBufferData", Diligent::SHADER_TYPE::SHADER_TYPE_VS_PS, maxDirectionalLights, maxDirectionalLights},
-                StructuredBufferResourceDesc{"PointLightBufferData",       Diligent::SHADER_TYPE::SHADER_TYPE_VS_PS, maxPointLights,       maxPointLights},
-                StructuredBufferResourceDesc{"SpotLightBufferData",        Diligent::SHADER_TYPE::SHADER_TYPE_VS_PS, maxSpotLights,        maxSpotLights},
-                StructuredBufferResourceDesc{"MaterialBufferData",         Diligent::SHADER_TYPE::SHADER_TYPE_PIXEL, maxMeshRenderers,     initialMaterialSizeHint},
-                TextureBufferResourceDesc{"TextureBufferData",             Diligent::SHADER_TYPE::SHADER_TYPE_PIXEL, maxTextures},
+                StructuredBufferResourceDesc{"TransformBufferData",        Diligent::SHADER_TYPE::SHADER_TYPE_VERTEX, maxMeshRenderers,     maxMeshRenderers / 2},
```

Comment:
what's with the /2 here? It seems a little arbitrary to me.

Reply:
Eh, I wasn't sure what to do here. Its the initial buffer size. It seemed unnecessary to allocate the max buffer, since we can resize. It would be silly to allocate 0, because there will certainly be some data. It also didn't seem like a meaningful value to set through the config. Maybe I'll make it a parameter and pass it in. It should probably be set the same as initialMaterialSizeHint for now.